test.cmake

```diff
@@ -35,6 +35,11 @@ IF(NOT DEFINED DISABLE_TESTS)
   SET(DISABLE_TESTS OFF)
 ENDIF(NOT DEFINED DISABLE_TESTS)
 
+# Active test suites if BUILD_TESTING is ON
+IF(BUILD_TESTING)
+  SET(DISABLE_TESTS OFF)
+ENDIF(BUILD_TESTING)
```

Comment:
`BUILD_TESTING` is a CMake variable, right ?

I would either do:
```cmake
IF(NOT DEFINED DISABLE_TESTS)
  IF(${BUILD_TESTING})
    SET(DISABLE_TESTS OFF)
  ELSE()
    SET(DISABLE_TESTS ON)
  ENDIF()
ENDIF(NOT DEFINED DISABLE_TESTS)
```
or
```cmake
IF(DEFINED DISABLE_TESTS)
  MESSAGE(AUTHOR_WARNING "DISABLE_TESTS is deprecated. Use BUILD_TESTING instead.")
  IF(${DISABLE_TESTS})
    SET(BUILD_TESTING OFF)
  ELSE()
    SET(BUILD_TESTING ON)
  ENDIF()
ENDIF(DEFINED DISABLE_TESTS)
```

Reply:
I changed the files to reflect your proposition.
Thanks for your review.